src/plugins/telemetryTable/components/TableComponent.vue

```diff
@@ -595,23 +596,35 @@ export default {
     },
     handleConfigurationChanges(changes) {
       const { rowLimit, telemetryMode, persistModeChange } = changes;
```

Comment:
If rowLimit shrinks, do we have to re-request?

Reply:
SOLID question...

Reply:
For now yes.